cli/pcluster/utils.py

```diff
@@ -166,9 +166,16 @@ def get_supported_features(region, feature):
     try:
         features = _get_json_from_s3(region, "features/feature_whitelist.json")
         supported_features = features.get("Features").get(feature)
-    except (ValueError, ClientError, KeyError):
+    except (ValueError, ClientError, KeyError) as e:
+        if type(e) is ClientError:
+            code = e.response.get("Error").get("Code")
+            message = e.response.get("Error").get("Message")
```

Comment:
you could avoid to define the message variable since we are using it only if `code == InvalidAccessKeyId`